api/src/pcapi/routes/adage_iframe/logs.py

```diff
@@ -46,7 +46,7 @@ def log_offer_details_button_click(
     body: serialization.StockIdBody,
 ) -> None:
     educational_utils.log_information_for_data_purpose(
-        event_name="OfferDetailButtonClick",
+        event_name="RequestDetailButtonClick",
```

Comment:
Idem pour le nom ici `RequestDetailButtonClick` : on ne comprend pas vraiment à quel évenement ça correpond. Pour rappel la fonction sert à logger la fermeture de la pop-in de demande d'offre réservable (CollectiveRequest). 

On pourrait donc avoir un nom plus cohérent comme `RequestPopinDismiss` ou `CollectiveRequestPopinDismiss`